repository/src/main/java/ca/bc/gov/repository/bcsc/BcscAuthRepo.kt

```diff
@@ -104,9 +113,7 @@ class BcscAuthRepo(
     * Check for logged in session
     * */
     suspend fun checkLogin(): Boolean {
```

Comment:
rename this method to isSessionActive or checkSession

Reply:
Done